scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -414,87 +404,88 @@ head( rowData(merged_sce) )
 
 ## Integration
 
+So far, we've created a `merged_sce` object which is (almost!) ready for integration.
 
-So far, we've created a `merged_sce` object which is (almost!) ready to integrate.
-
-The integration methods we'll be using here actually perform batch correction on a reduced dimension representation of the normalized gene expression values, specifically the PCA.
-The main reason for using these reduced dimensions is efficiency.
+The integration methods we'll be using here actually perform batch correction on a reduced dimension representation of the normalized gene expression values.
+`fastMNN` and `harmony` specifically use PCA for this, but be aware that different integration methods may use other kinds of reduced dimensions!
+The main reason for using reduced dimensions is efficiency.
 
 You'll notice that the merged SCE object object already contains PCA and UMAP reduced dimensions, which were calculated during our pre-processing:
 
-```{r merged_sce view pca}
+```{r merged sce view pca}
 reducedDimNames(merged_sce)
 ```
 
-These represent the original dimension reductions that were performed on _each individual library_, but we actually need to calculate PCA (and UMAP for visualization) from the merged object directly.
+These represent the original dimension reductions that were performed on _each individual SCE_ before merging, but we actually need to calculate PCA (and UMAP for visualization) from the merged object directly.
 
-Why can't we use the library-specific PCA? 
-Part of the PCA calculation itself involves scaling the raw data to center the mean.
-When this is performed on each library individually, all libraries are separately centered, which leads to the data "overlapping" in space.
-To see this, let's look at the UMAP for individual libraries:
+Why can't we use the sample-specific PCA and UMAP matrices? 
+Part of these calculations themselves involves scaling the raw data to center the mean.
+When this is performed on each sample individually, all samples are separately centered, which leads to the data "overlapping" in space while also influencing coordinates overall.
```

Comment:
I want to emphasize this a bit more? I'm not totally sure of the wording, but something like this?
```suggestion
When this is performed on each sample individually, all samples are separately centered, which leads to the data "overlapping" in space.
The mapping function from the original data to the reduced dimensions will also be separately calculated, so the new coordinates won't have consistent meanings across samples.
```